metamorph/src/test/java/org/metafacture/metamorph/collectors/TuplesTest.java

```diff
@@ -35,127 +32,125 @@
-    ordered.verify(receiver).literal("product", "cCY");
-    ordered.verify(receiver).literal("product", "dCY");
-    ordered.verify(receiver).literal("product", "cDY");
-    ordered.verify(receiver).literal("product", "dDY");
-    ordered.verify(receiver).endRecord();
-    ordered.verifyNoMoreInteractions();
-  }
-
-  @Test
-  public void shouldOnlyEmitTriplesWithMoreThanMinNValues() {
-    metamorph = InlineMorph.in(this)
-        .with("<rules>")
-        .with("  <tuples name='product' minN='3'>")
-        .with("    <data source='1' />")
-        .with("    <data source='3' />")
-        .with("    <data source='2' />")
-        .with("  </tuples>")
-        .with("</rules>")
-        .createConnectedTo(receiver);
-
-    metamorph.startRecord("1");
-    metamorph.literal("1", "a");
-    metamorph.literal("1", "b");
-    metamorph.literal("2", "A");
-    metamorph.literal("2", "B");
-    metamorph.endRecord();
-    metamorph.startRecord("2");
-    metamorph.literal("3", "X");
-    metamorph.literal("1", "c");
-    metamorph.literal("1", "d");
-    metamorph.literal("2", "C");
-    metamorph.literal("3", "Y");
-    metamorph.literal("2", "D");
-    metamorph.endRecord();
-
-    final InOrder ordered = inOrder(receiver);
-    ordered.verify(receiver).startRecord("1");
-    ordered.verify(receiver).endRecord();
-    ordered.verify(receiver).startRecord("2");
-    ordered.verify(receiver).literal("product", "cCX");
-    ordered.verify(receiver).literal("product", "dCX");
-    ordered.verify(receiver).literal("product", "cDX");
-    ordered.verify(receiver).literal("product", "dDX");
-    ordered.verify(receiver).literal("product", "cCY");
-    ordered.verify(receiver).literal("product", "dCY");
-    ordered.verify(receiver).literal("product", "cDY");
-    ordered.verify(receiver).literal("product", "dDY");
-    ordered.verify(receiver).endRecord();
-    ordered.verifyNoMoreInteractions();
-  }
-
-  @Test
-  public void shouldEmitTuplesWithMinNIfNotAllStatementsFired() {
-    metamorph = InlineMorph.in(this)
-        .with("<rules>")
-        .with("  <tuples name='product' minN='1'>")
-        .with("    <data source='1' />")
-        .with("    <data source='3' />")
-        .with("    <data source='2' />")
-        .with("  </tuples>")
-        .with("</rules>")
-        .createConnectedTo(receiver);
-
-    metamorph.startRecord("1");
-    metamorph.literal("1", "a");
-    metamorph.literal("1", "b");
-    metamorph.endRecord();
-
-    final InOrder ordered = inOrder(receiver);
-    ordered.verify(receiver).startRecord("1");
-    ordered.verify(receiver).literal("product", "a");
-    ordered.verify(receiver).literal("product", "b");
-    ordered.verify(receiver).endRecord();
-    ordered.verifyNoMoreInteractions();
-  }
+    @Rule
+    public final MockitoRule mockitoRule = MockitoJUnit.rule();
+
+    @Mock
+    private StreamReceiver receiver;
+
+    @Test
+    public void shouldEmitTwoandThreeTuples() {
+        assertMorph(receiver,
```

Comment:
s#TwoandTh#TwoAndTh#g

Reply:
Well, it's not my typo... but I've fixed it ;)